cypress/integration/ProcessorTest.ts

```diff
@@ -0,0 +1,449 @@
+describe("Processor Test", () => {
+  /**
+   * This test suite verifies that the user can create all types of processors.
+   * Because it is running with mocked data we have to assert all data immediately (no reload of the page).
+   */
+  describe("Create a Processor", () => {
+    beforeEach(() => {
+      cy.visit("/instance/830c8f0d-c677-492f-8d7e-0f81893fbba6");
+      cy.ouiaId("loading-table", "PF4/Card", { timeout: 30000 }).should(
+        "not.exist"
+      );
+    });
+
+    it("Sink processor", () => {
+      const processorName: string = "Sink processor";
+      const action = [
+        "Send to Slack",
+        "dev-action",
+        "https://test.app.com/item",
+      ];
+      const filters = [
+        ["data.name", "processor.StringEquals", "John"],
+        ["data.surname", "processor.StringEquals", "White"],
```

Comment:
juts double checking, I think in some other PR there was a discussion `processor.StringEquals` is a consequence of some missing translation? is `processor.StringEquals` string constant still expected at this line?